iothub/device/tests/Transport/Amqp/ClientWebSocketTransportTests.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Net;
+using System.Net.WebSockets;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Microsoft.Azure.Amqp;
+using Microsoft.Azure.Amqp.Transport;
+using Microsoft.Azure.Devices.Client.HsmAuthentication.Transport;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.Azure.Devices.Client.Test.Transport
+{
+    [TestClass]
+    [TestCategory("Unit")]
+    public class ClientWebSocketTransportTests
```

Comment:
Not sure if we can add more tests for ClientWebSocket. Most methods require open connection.